run/GCHP/ExtData.rc.templates/ExtData.rc.TransportTracers

```diff
@@ -156,14 +156,17 @@ EDGAR_CO_25_FFF  kg/m2/s N Y F%y4-01-01T00:00:00 none none emi_co  ./HcoDir/EDGA
 #==============================================================================
 # --- CEDS (CEDSv2) ---
```

Comment:
Change this to CEDS_01x01

Reply:
Now resolved in 1ef383b65d4a54249b1421d5fa3049cfc7ddb912